snapshot-replicator/functions/create_snapshot.py

```diff
@@ -1,14 +1,11 @@
 import boto3  
-import botocore  
 import datetime  
 import re
 import os
-import datetime
 
-#SOURCE_REGION = os.environ['SOURCE_REGION']
-SOURCE_REGION = "eu-west-1"  
-iam = boto3.client('iam')  
-instances = ["frisket-production-api-rds01"]
+SOURCE_REGION = os.environ['SOURCE_REGION']
+instances = os.environ['DB_INSTANCES']
```

Comment:
can you update your code to either use all caps for env variables or all lowercase? atm its mixed with eg `SOURCE_REGION` and `instances`